op-batcher/batcher/driver.go

```diff
@@ -535,6 +535,29 @@ func (l *BatchSubmitter) receiptsLoop(wg *sync.WaitGroup, receiptsCh chan txmgr.
 	l.Log.Info("receiptsLoop returning")
 }
 
+func (l *BatchSubmitter) setMaxDASizeOnEndpoint(ctx context.Context, endpoint string, client *rpc.Client, maxTxSize, maxBlockSize uint64) (bool, error) {
```

Comment:
why don't you just return only an `error` and encode success with a `nil` error? this would be more idiomatic.